Adyen/Model/Checkout/CheckoutUtilityRequest.cs

```diff
@@ -0,0 +1,143 @@
+#region Licence
+
+// 
+//                        ######
+//                        ######
+//  ############    ####( ######  #####. ######  ############   ############
+//  #############  #####( ######  #####. ######  #############  #############
+//         ######  #####( ######  #####. ######  #####  ######  #####  ######
+//  ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+//  ###### ######  #####( ######  #####. ######  #####          #####  ######
+//  #############  #############  #############  #############  #####  ######
+//   ############   ############  #############   ############  #####  ######
+//                                       ######
+//                                #############
+//                                ############
+// 
+//  Adyen Dotnet API Library
+// 
+//  Copyright (c) 2020 Adyen B.V.
+//  This file is open source and available under the MIT license.
+//  See the LICENSE file for more info.
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using Adyen.Util;
+using Newtonsoft.Json;
+
+namespace Adyen.Model.Checkout
+{
+    /// <summary>
+    /// CheckoutUtilityRequest
```

Comment:
I think Checkout moved the originKeys endpoint from the CheckoutUtility service to the Checkout service: https://docs.adyen.com/api-explorer/#/CheckoutService/v66/post/originKeys

Should we remove the Checkout Utility service and use this class instead to retrieve origin keys?

Reply:
Switched to checkout using CheckoutUtilityRequest and CheckoutUtilityResponse instead CheckoutUtility